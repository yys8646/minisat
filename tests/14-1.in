14
..11...0.1..1.
....0.......0.
11.....1...0..
1....10..1.00.
.11.0...0....0
..........1...
..00...1.1...0
.0........0...
..1..11....1..
...0...0..0...
....1...0..11.
..............
.1.......1....
...1....0...10
