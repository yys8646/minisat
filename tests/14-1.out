14
00110110010110
01100110100101
11001001101010
10010101011001
01100110010110
10011010101001
11001001011010
00110101100101
00110110101100
11001010010011
10101001010110
00110101101001
01001010110101
11011001001010
